package.json

```diff
@@ -30,13 +27,7 @@
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
-  "dependencies": {
-    "@nestjs/graphql": "^9.1.2",
-    "apollo-server-core": "^3.5.0",
-    "graphql": "15",
```

Comment:
These aren't referenced anywhere in the code.  I'm guessing they were included as they are peer dependencies of @nesths/graphql.  However, it is not necessary to include them here.  The main app package.json will include all the peer dependencies

Reply:
I would prefer updating the dependencies in another pull request.

Reply:
You are right.  I'll submit a second one after this one is complete.